x-pack/plugins/security_solution/public/resolver/models/indexed_process_tree/isometric_taxi_layout.ts

```diff
@@ -525,7 +543,7 @@ export function translated(model: IsometricTaxiLayout, translation: Vector2): Is
       ])
     ),
     edgeLineSegments: model.edgeLineSegments.map(({ points, metadata }) => ({
-      points: points.map((point) => vector2.add(point, translation)),
+      points: [vector2.add(points[0], translation), vector2.add(points[1], translation)],
```

Comment:
This code does the same thing as before, but I changed the type from `Vector2[]` to `[Vector2, Vector2]` and `.map` will not return a tuple even when called on a tuple.

Reply:
will return a tuple?

Reply:
`points` is a tuple but `.map` always returns an array type. So I wrote the tuple out explicitly.